client/js/uploader.api.js

```diff
@@ -523,7 +523,7 @@
                 self = this,
                 retVal;
 
-            retVal = this._options.showConfirm(confirmMessage);
+            retVal = this._options.showConfirm(confirmMessage, id, uuid);
```

Comment:
Please remove `uuid` since this can be obtained via the `getUuid()` API method. Also please update the docs after this change.